Platform/AlderlakeBoardPkg/Library/Stage1BBoardInitLib/Stage1BBoardInitLib.h

```diff
@@ -46,7 +46,6 @@
 #include <Library/PchInfoLib.h>
 #include <Library/VariableLib.h>
 #include <Library/GpioLib.h>
-#include <Library/TopSwapLib.h>
```

Comment:
This library is used by FwUpdate/Resiliency related features. what's the reason to remove it?

Reply:
I donot see any build issues. I enabled the resilency flag in board config and compiled. No issues flagged by this removal.

Reply:
Linux build tagged this as an issue saying duplicate declarations. Hence had to remove.

Reply:
Funbction Setbootpartition() is used from it.
@sean-m-mcginn Can you please help review if this change is OK?